ISSUE_TRIAGE_PROCESS.rst

```diff
@@ -151,11 +154,7 @@ to be able to reproduce the issue. Typically, this may require a response to the
 issue creator asking for more information, with the issue then being tagged with
 the label ``pending-response``.
 Also, during this stage, additional labels may be added to the issue to help
-classification and triage, such as ``reported_version`` and ``area``.
-
-Occasionally an issue may require a larger discussion among the Airflow PMC or
-the developer mailing list. This issue may then be tagged with the
-``needs:discussion`` label.
+classification and triage, such as ``affected_version`` and ``area``.
```

Comment:
I'll do the rename `reported_version` -> `affected_version` after merge

Reply:
We should decide (and then document here) if this label will be used to tag _all_ affected versions, or just the latest/most current affected version. The former seems like it'd be interesting data but lots of paperwork, the latter seems to be what we're most interested in (knowing how current an issue is). What do folks think?

Reply:
That could be useful if we would have release fixes to older versions as well but once we bump feature release then we don't fix past versions no matter what.
That said - the goal of labeling is to help contributors to navigate through the issues. If someone finds this helpful and it has merit maybe it's worth the overhead.

Currently we use policy of only 1 reported version label.

Reply:
Yeah, agreed on both counts. I honestly don't feel _too_ strongly either way. As long as we document what our policy is in this rst.